openerp/addons/openupgrade_records/lib/apriori.py

```diff
@@ -4,6 +4,7 @@
 
 renamed_modules = {
     'portal_claim': 'website_crm_claim',
+    'portal_project_issue': 'website_project_issue',
```

Comment:
This renaming is dangerous, as if you don't have website module installed, this will install all the website stack. You should merge it with `project_issue` instead.

Reply:
Done